_std/defines/airlock.dm

```diff
@@ -0,0 +1,13 @@
+//check defines/accessLevels.dm for access levels! this is just airlock specific stuff
```

Comment:
better done in a `door.dm` file

Reply:
i dont want to merge access levels and doors, because access levels are also used for computers and stuff think?? ill rename the file tho!